addon-test-support/-private/async-iterator.js

```diff
@@ -41,7 +43,9 @@ export default class AsyncIterator {
    */
   handleResponse(response) {
     if (this._waiting === false) {
-      throw new Error(this.toString() + ' Was not expecting a response, but got a response');
+      throw new Error(
+        this.toString() + ' Was not expecting a response, but got a response'
```

Comment:
lets use template string here
`${this.toString()} Was not expecting a response, but got a response.`